sky/spot/spot_utils.py

```diff
@@ -749,13 +749,13 @@ def is_spot_controller_up(
           identity.
     """
     try:
-        # Set force_refresh=False to make sure the refresh only happens when the
+        # Set force_refresh=None to make sure the refresh only happens when the
```

Comment:
Q: why is setting it to None the same as “refresh only when the controller is init/up”?

Reply:
It is because the spot controller will always have the autostop setup, which will trigger the refresh for both init and up cases for the controller.